Source/ChocolateyGui.Common.Windows/Services/PackageArgumentsService.cs

```diff
@@ -0,0 +1,64 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright company="Chocolatey" file="PackageArgumentsService.cs">
+//   Copyright 2017 - Present Chocolatey Software, LLC
+//   Copyright 2014 - 2017 Rob Reynolds, the maintainers of Chocolatey, and RealDimensions Software, LLC
+// </copyright>
+// --
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+using chocolatey;
+
+namespace ChocolateyGui.Common.Windows.Services
+{
+    public class PackageArgumentsService : IPackageArgumentsService
+    {
+        private static readonly byte[] EntropyBytes = Encoding.UTF8.GetBytes("Chocolatey");
+
+        public string Decrypt(string id, string version)
+        {
+            var packageFolder = "{0}.{1}";
+            var packageId = id;
+            var packageVersion = version;
+            var chocolateyPath = @"C:\ProgramData\chocolatey\.chocolatey";
+            var argumentsPath = Path.Combine(chocolateyPath, packageFolder.format_with(id, version));
+            var argumentsFile = Path.Combine(argumentsPath, ".arguments");
+
+            // Get the arguments decrypted in here and return them
+            try
+            {
+                if (File.Exists(argumentsFile))
+                {
+                    string fileName = argumentsFile;
+                    var encryptedBytes = Convert.FromBase64String(File.ReadAllText(fileName));
+                    var decryptedBytes = ProtectedData.Unprotect(encryptedBytes, EntropyBytes, DataProtectionScope.LocalMachine);

```

Comment:
Thinking about it, I think we can make this a little simpler as well..  Chocolatey.Lib already has a service for getting this information: https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure/cryptography/DefaultEncryptionUtility.cs so we should be able to call into this.  That way, Chocolatey GUI is doing things in exactly the same way as Chocolatey itself.